.dockerenvrc

```diff
@@ -1,3 +1,4 @@
 NODE_ENV='production'
 MONGO_ADDRESS='mongodb://mongo:27017/volume'
-SERVER_ADDRESS='http://volume-backend.cornellappdev.com'
\ No newline at end of file
+SERVER_ADDRESS='http://volume-backend.cornellappdev.com'
+IMAGE_PATH='https://raw.githubusercontent.com/cuappdev/assets/master/volume'
```

Comment:
super nit-picky: For consistent variable naming, I think `IMAGE_ADDRESS` is more fitting?

Reply:
Agreed and changed